### DISCUSSION TITLE
Problem with tokengen

### DISCUSSION THREAD

**Discussion by killianvervelle** [hyperledger/fabric-samples#1214] [General]
Hi there, silly question but how did you get this command to work :  ```go install github.com/hyperledger-labs/fabric-token-sdk/cmd/tokengen@0.3.0.```
I get this error :
``` # github.com/hyperledger/fabric-private-chaincode/internal/crypto
../go/pkg/mod/github.com/hyperledger/fabric-private-chaincode@v0.0.0-20210907122433-d56466264e4d/internal/crypto/crypto.go:24:10: undefined: GoCrypto```

**Comment by denyeart** [comment]
`go install github.com/hyperledger-labs/fabric-token-sdk/cmd/tokengen@v0.3.0` works for me. Note the extra `v`.

**Comment by killianvervelle** [comment]
Thank you very much for your reply. Yes i have added the -v, still getting the same error "Undefined GoCrypto....". 
Can you please share your go version ? 
also, do you install it in Ubuntu 22.04 ?
Do I have to install it in a special location inside fabric-samples?

**Comment by killianvervelle** [comment]
Also, do I need to install any other dependencies ?

**Comment by killianvervelle** [comment]
I followed the instruction inside the token-sdk repo
